### DISCUSSION TITLE
Livewire V3 not working with laravel MIX

### DISCUSSION THREAD

**Discussion by iislamtaleb** [livewire/livewire#7854] [7. Bugs (archived)]
### Livewire version

v3.4

### Laravel version

8.1.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

I am working on a project and needed to install the Mix plugin alongside Vite. However, I encountered an issue when creating new Livewire components after installing Laravel Mix, Livewire didn't work

after that I have removed Mix and Install Vite again and I come to this issue

![Capture d'écran 2024-01-30 224317](https://github.com/livewire/livewire/assets/148808316/9b5c907f-2f66-4acc-ad1d-d4ea21b80c2c)
![Capture d'écran 2024-01-30 224344](https://github.com/livewire/livewire/assets/148808316/dc5ab98a-dd75-4daa-a94d-9a3b4ae504aa)
![Capture d'écran 2024-01-30 224402](https://github.com/livewire/livewire/assets/148808316/cf6b4320-8b8a-434f-93f4-feb1ba68d195)

### Repository or Wirebox URL

**

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@iislamtaleb please don't open multiple discussions for the same issue. I have already replied on your other discussion https://github.com/livewire/livewire/discussions/7848#discussioncomment-8312321 so I'm going to close this one.
